astropy/io/votable/tests/test_vo.py

```diff
@@ -947,15 +947,10 @@ def test_build_from_scratch(tmp_path):
     )
 
 
-def test_validate(test_path_object=False):
-    """
-    test_path_object is needed for test below ``test_validate_path_object``
-    so that file could be passed as pathlib.Path object.
-    """
+@pytest.mark.parametrize("fpath_transform", [lambda _: _, lambda f: Path(f)])
```

Comment:
```suggestion
@pytest.mark.parametrize("path_type", [str, Path])
```
This way the IDs `pytest` assigns to the parametrizations are much better.